spinach/worker.py

```diff
@@ -18,7 +17,7 @@
 logger = getLogger(__name__)
 
 
-class BaseWorkers(ABC):
+class BaseWorkers:
```

Comment:
This was defined as an abstract base class, but it provided zero abstract methods.

Quietly upgrading it to a "plain" base class seems to be the thing to do here.  But I could be wrong...